src/datasets/domain/repositories/IDatasetsRepository.ts

```diff
@@ -32,4 +32,5 @@ export interface IDatasetsRepository {
     datasetMetadataBlocks: MetadataBlock[],
     collectionId: string
   ): Promise<CreatedDatasetIdentifiers>
+  publishDataset(datasetId: number | string, versionUpdateType: 'major' | 'minor'): Promise<void>
```

Comment:
Why not use `versionUpdateType: VersionUpdateType` instead of `versionUpdateType: 'major' | 'minor'`?

Reply:
I probably forgot. Thanks! I made the change